### PR TITLE
Log cpu usage periodically

### PR DESCRIPTION
**Pull request by Megamouse**

### CODE REVIEW

**Review by VelocityRa — COMMENTED** [review]
"perf sensor" is a bit awkward naming imo, i'd change to "perf monitor"

**Comment by Megamouse** [comment]
Yeah that was the word i was looking for 😂 thx

**Comment by clienthax** [comment]
Can we get a toggle?

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -1115,6 +1119,9 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		// Initialize progress dialog
 		g_fxo->init<named_thread<progress_dialog_server>>();
 
+		// Initialize performance sensor
```
Comment (sampletext32) [inline rpcs3/Emu/System.cpp:1122]: Comment mismatches the class name now. 
I doubt it's really necessary, because the code is self-explanatory.

**Pull request merged**